src/_posts/hands-on-i.md

```diff
@@ -0,0 +1,149 @@
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/tHP2kX6aAZM?rel=0&amp;showinfo=0" frameborder="0" allowfullscreen></iframe>
+
+The required steps are as follow:
+
+  1. With a hot air gun, soften the soldering around the chip.
+  2. With the help of a suction pad, separate the chip from the circuit board.
+  3. Carefully place the chip on the table, to avoid ruining the chip pins.
+
+I chose to implement one step at a time without realistic models in the beginning. Once I am able to recreate the complete process, I can think about adding different levels of detail to the demo.
+
+But before providing specific simulations and behaviors, we need some basic hand-based interaction.
+
+## Setting the scene up
+
+Let’s start looking at the code. The first thing I needed was a scene containing all the elements I wanted to work with: the air gun, the suction pad, the chip, the soldering around the chip, and the circuit board. Of course I needed a representation of my own hands and some environmental elements such as a sky, a floor and of course a table!
+
+This is the initial code for the scene:
+
+```html
+<a-scene>
+  <!-- World -->
+  <a-sky color="#333"></a-sky>
+  <a-plane rotation="-90 0 0"
+           width="15" height="15" color="#777">
+  </a-plane>
+  <!-- Workspace -->
+  <a-entity id="workspace" position="0 0.8 -0.8">
+    <a-box id="workbench" position="0 0 0"
+           width="2" depth="1" height="0.02" color="#97b7d5">
+    </a-box>
+    <a-box id="circuitboard" position="0 0.0115 0.4"
+           width="0.05" depth="0.05" height="0.003" color="#008000">
+      <a-box id="chip" position="0 0.002 0.005"
+             width="0.02" depth="0.02" height="0.001" color="#000">
+      </a-box>
+    </a-box>
+    <a-cylinder id="suction-pad"
+                position="-0.2 0.015 0.286" rotation="-90 0 0"
+                height="0.1" radius="0.005" color="red">
+    </a-cylinder>
+    <a-cylinder id="hot-air-gun"
+                position="0.3 0.02 0.286" rotation="-90 0 0"
+                height="0.15" radius="0.01" color="blue">
+    </a-cylinder>
+  </a-entity>
+  <!-- Operator -->
+  <a-camera position="0 0 0" user-height="1.3"></a-camera>
+  <a-entity id="left-hand" hand-controls="left"></a-entity>
+  <a-entity id="right-hand" hand-controls="right"></a-entity>
+</a-scene>
+```
+
+When working with A-Frame, all the scene objects are contained inside the `a-scene` element. A-Frame provides some special tags to achieve common tasks such as adding a sky dome or a camera into the virtual space and it also has primitives available such as boxes, planes, or cylinders.
```

Comment:
Is this paragraph redundant / not useful for someone just interested on learning about hand interaction? This feels more like an intermediate article?